paper/paper.md

```diff
@@ -0,0 +1,137 @@
+Although this is arguably the best package to achieve interaction between both languages, it has many disadvantages if one wants to use it just to load RData datasets.
+In the first place, the package requires an R installation, as it relies in launching an R interpreter and communicating with it.
+Secondly, launching R just to load data is inefficient, both in time and memory.
+Finally, this package inherits the GPL license from the R language, which is not compatible with most Python packages, typically released under more permissive licenses.
+
+The package `pyreadr` [@fajardo_2024_pyreadr] also provides functionality to read and write some R datasets.
+It relies in the C library `librdata` in order to perform the parsing of the RData format.
+This adds an additional dependency from C building tools, and requires that the package is compiled for all the desired operating systems.
+Moreover, this package is limited by the functionalities available in `librdata`, which at the moment of writing
+does not include the parsing of common objects such as R lists and S4 objects.
+The license can also be a problem, as it is part of the GPL family and does not allow commercial use.
+
+As existing solutions were unsuitable for our needs, the package `rdata` was developed to parse data in the RData format.
+This is a small, extensible, efficient, and very complete implementation in pure Python of a RData parser, that is able to read and convert most datasets in the CRAN repository to equivalent Python objects, such as the built-in types of The Python Standard Library, NumPy arrays [@harris+_2020_numpy], or Pandas dataframes [@mckinney_2010_pandas; @pandasdevelopmentteam_2024_pandasdev].
+It has a permissive license and can be extended to support additional conversions from custom R classes.
+
+The package `rdata` has been designed as a pure Python package with minimal dependencies, so that it can be easily integrated inside other libraries and applications.
+It currently powers the functionality offered in the `scikit-datasets` package [@diaz-vico+ramos-carreno_2023_scikitdatasets] for loading datasets from the CRAN repository of R packages.
+This functionality is used for fetching the functional datasets provided in the `scikit-fda` library [@ramos-carreno+_2024_scikitfda], whose development was the main reason for the creation of the `rdata` package itself.
+
+# Features
+
+The package `rdata` is intended to be both flexible and easy to use.
+In order to be flexible, the parsing of the RData format and the conversion of the parsed structures to appropriate Python objects have been splitted in two steps.
+This allows advanced users to perform custom conversions without losing information.
+Most users, however, will want to use the default conversion routine, which attempts to convert data
+to a standard Python representation which preserves most part of the information.
+
+```python
+import rdata
+
+converted = rdata.read_rda("dataset.rda")
+converted
```

Comment:
```suggestion
```
I'd suggest to remove this line (or use `print(converted)` instead if printing is intended.